dev/openchpl_preload.sql

```diff
@@ -1033,9 +1033,11 @@ INSERT INTO openchpl.test_tool(name, last_modified_user, retired) VALUES
 ('Electronic Prescribing', -1, false),
 ('HL7 CDA National Health Care Surveys Validator', -1, false),
 ('Edge Testing Tool', -1, false),
-('CDC''s NHSN CDA Validator', -1, false),
+('CDC''s NHSN CDA Validator', -1, true),
 ('NCQA ONC Health IT Testing', -1, false),
 ('HIMSS Immunization Integration Program', -1, false);
```

Comment:
Did you run the `reset-test-only.sh` script? It should have given you an error because this line ends with a semi-colon, not a comma